Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.8-slim-buster
+FROM python:3.9-slim
 WORKDIR /usr/src/app
 COPY requirements_old.txt ./
 RUN pip3 install --upgrade pip
 RUN pip3 install --no-cache-dir -r requirements.txt
```

Comment:
I see the problem here is that we copy `requirements_old.txt` and then use `requirements.txt`

Reply:
Ok